src/reaper_osara.cpp

```diff
@@ -655,25 +669,47 @@ void postGoToSpecificMarker(int command) {
 	}
 }
 
-void postChangeTrackVolume(MediaTrack* track) {
-	double volume = 0.0;
-	if ( !GetTrackUIVolPan(track, &volume, NULL) )
-		return;
+void postChangeVolumeH(double volume, int command, char* commandMessage) {
 	ostringstream s;
+	if(lastCommand != command) 
+		s << commandMessage;
 	s << fixed << setprecision(2);
 	s << VAL2DB(volume);
 	outputMessage(s);
 }
 
 void postChangeTrackVolume(int command) {
 	MediaTrack* track = GetLastTouchedTrack();
-	if (!track)
+	double volume = 0.0;
+	if ( !GetTrackUIVolPan(track, &volume, NULL) )
 		return;
-	postChangeTrackVolume(track);
+	postChangeVolumeH(volume, command, "Track ");
 }
 
 void postChangeMasterTrackVolume(int command) {
-	postChangeTrackVolume(GetMasterTrack(0));
+	MediaTrack* track = GetMasterTrack(0);
+	double volume = 0.0;
+	if ( !GetTrackUIVolPan(track, &volume, NULL) )
+		return;
+	postChangeVolumeH(volume, command, "Master ");
+}
+
+void postChangeItemVolume(int command) {
+	MediaItem* item = getItemWithFocus();
+	if(!item)
+		return;
+	double volume = GetMediaItemInfo_Value(item, "D_VOL");
+	postChangeVolumeH(volume, command, "Item ");
+}
+
+void postChangeTakeVolume(int command) {
+	MediaItem* item = getItemWithFocus();
+	if(!item)
+		return;
+	MediaItem_Take* take = GetActiveTake(item);
```

Comment:
An empty item will have no take, so we have to check for that lest we crash.
```suggestion
	MediaItem_Take* take = GetActiveTake(item);
	if (!take) {
		return;
	}
```